src/bin/build.rs

```diff
@@ -16,10 +17,6 @@ pub fn make_subcommand<'a, 'b>() -> App<'a, 'b> {
             "--no-create 'Will not create non-existent files linked from SUMMARY.md'",
         )
         .arg_from_usage(
-            "--curly-quotes 'Convert straight quotes to curly quotes, except for those \
-             that occur in code blocks and code spans'",
-        )
-        .arg_from_usage(
             "[dir] 'A directory for your book{n}(Defaults to Current Directory \
```

Comment:
Why is the "curly quote" option removed?

Reply:
I think I removed it because something like that belongs more in the config file than as a command line argument. My thoughts were that options to alter the generated output and which usually stay the same belong to go in the config file, while options that alter runtime behaviour (e.g. `--open` or `--no-create`) are command line arguments... Does that sound logical?

Reply:
That sounds reasonable to me, but I am not sure about the intent behind the `curly quote` option.